frontend_tests/run-casper

```diff
@@ -71,7 +71,11 @@ subprocess.check_call(['mkdir', '-p', 'var/casper'])
 
 subprocess.check_call(['rm', '-f'] + glob.glob('var/casper/casper-failure*.png'))
 
-log = open('var/casper/server.log', 'w')
+if os.path.getsize('var/casper/server.log') < 100000:
```

Comment:
tests fail because you need to check `os.path.exists('var/casper/server.log')`.  Also I'd recommend moving `var/casper/server.log` into a variable since it's now going to appear 4 times here.